tests/integration/targets/fusion_hap/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+- name: Create new Linux host access policy
+  purestorage.fusion.fusion_hap:
+    name: hap_foo
+    personality: linux
+    iqn: "iqn.2005-03.com.RedHat:linux-host1"
+  environment: "{{ test_env }}"
```

Comment:
Why there are checks `result is success` after some tasks. But for example here there is no such check

Reply:
I used them where I could not get the list of the resources from fusion_info. I will remove them where they are not needed.

Reply:
Actually, I will add them in all task, it's a good check that the task was successful

Reply:
Added the validation tasks where applicable.